cypress/integration/basic/smoke-test.js

```diff
@@ -1,125 +1,217 @@
   before(function () {
     cy.visit('/?planetWizard=true')
     cy.wait(3000)
+    cy.waitForSplashscreen()
   })
 
+  let bottom = new BottomContainer()
+  let top = new TopContainer()
+  let planetWizard = new PlanetWizard()
+
   context('Loading screen and initialization of app', () => {
-    it('drags and disorients the planet', () => {
-      cy.get('.planet-wizard-overlay > :nth-child(1)').click()
-      // cy.get('.interaction-selector > :nth-child(3)').click()
-      cy.wait(7000)
-      cy.get('canvas').eq(1)
-        .trigger('mousedown', { force: true }, { which: 1, pageX: 500, pageY: 300 })
-        .trigger('mousemove', { force: true }, { which: 1, pageX: 350, pageY: 350 })
-        .trigger('mouseup', { force: true })
-    })
-    it('Makes sure splash screen renders before page loads', () => {
-      // check splashscreen shows up
+    it('Makes sure splash screen renders and disappears', () => {
+      cy.waitForSplashscreen()
     })
     it('verifies the logo', () => {
-      // check logo visibility
+      bottom.getBigLogo()
+        .should('exist')
+        .and('be.visible')
     })
     it('verifies refresh', () => {
-      // check visibility
-      // click plate num option
-      // refresh page
-      // check for plate num options
+      top.getRefresh()
+        .should('be.visible')
+      planetWizard.getAllPlateNumOptions()
+        .children()
+        .should('have.length', 5)
+      planetWizard.getPlateNumOption('2')
+        .should('be.visible')
+        .click({ force: true })
+      planetWizard.getAllPlateNumOptions()
+        .should('not.exist')
+      cy.wait(4000)
+      top.getRefresh()
+        .should('be.visible')
+        .click({ force: true })
+      cy.waitForSpinner()
+      planetWizard.getPlateNumOption('3')
+        .should('exist')
+        .and('be.visible')
     })
+
     it('verifies share', () => {
-      // checks visibility
-      // clicks share
-      // check for title "Share: Tectonic Explorer"
-      // check for link and html are present
-      // click close
+      top.getShare()
+        .should('be.visible')
+        .and('contain', 'Share')
+        .click({ force: true })
+      cy.waitForSpinner()
+      cy.get('[data-react-toolbox=dialog]').get('[data-react-toolbox=button]')
+        .should('contain', 'copy link')
+        .should('contain', 'copy html')
+        .should('contain', 'close')
+      cy.get('[data-react-toolbox=dialog]').get('[data-react-toolbox=button]').eq(2)
+        .click({ force: true })
     })
+
     it('verifies about', () => {
-      // verifies visibility
-      // clicks about
-      // checks for title About: Tectonic Explorer
+      top.getAbout()
+        .should('be.visible')
+        .and('contain', 'About')
+        .click({ force: true })
+      cy.get('section')
+        .should('contain', 'About: Tectonic Explorer')
+        .and('contain', 'Piotr Janik')
+      cy.get('body')
+        .click('left')
+      cy.wait(3000)
```

Comment:
Is it necessary at all? This test doesn't seem to have any more assertions after that.